.github/CODEOWNER

```diff
@@ -0,0 +1 @@
+* @elastic/platform-docs
```

Comment:
```suggestion
* @elastic/platform-docs

/docs/en/stack/ml @ml-docs
```

@szabosteve @chriscressman Let me know if there is a better Elastic team alias to use here.

Reply:
I went with @elastic/mlr-docs instead @elastic/ml-docs as `ml-docs` is a child of `mlr-docs`.